### DISCUSSION TITLE
remote desktop connection

### DISCUSSION THREAD

**Discussion by thisledome5** [amidaware/tacticalrmm#1867] [General]
Hello, A question on the RDP connection if someone may have knowledge.
When I initiate a connection to a machine that has RDP working on it and I connect remotely using RDP system separately, all it does is come up with a black screen and the disconnect button. nothing else.
Is there something I may have missed in settings? 

I use RDP to connect to this server regularly. It works perfectly on an alternate port using a secure SDWAN IP that the remote machine and the server are connected to. I can ping the server from the TRMM server on the SDWAN system. so it sees it. I set the alternate port on that right click on RDP. then initiate a connection using username and PW. it just shows a black screen with the disconnect top left. 

Any ideas here?

**Comment by silversword411** [comment]
Are you using the right words? RDP is a Microsoft proprietary remote control protocol. There is an optional web based RDP client mesh has, but that's not the default method of remote control in MeshCentral.

https://docs.tacticalrmm.com/mesh_integration/#take-control-connect-vs-rdp-connect
